[Bug] Unable to connect to MySQL after upgrade to version 1.5

A MySQL datasource on a self-hosted instance that was working before the upgrade to v1.5, was reported to fail repeatedly with no details of the error and no workaround. Following is the stacktrace:

```
  - [id: 0xcdbc3f5c, L:/10.10.108.204:50378 ! R:test.xxxxxxxxxxx.us-west-2.rds.amazonaws.com/10.10.10.10:3306] Error was received while reading the incoming data. The connection will be closed.           │
│ reactor.core.Exceptions$BubblingException: dev.miku.r2dbc.mysql.client.MySqlConnectionClosedException: Connection unexpectedly closed                                                                            │
│     at reactor.core.Exceptions.bubble(Exceptions.java:170)                                                                                                                                                       │https://codeshare.io/
│     at reactor.core.publisher.Operators.onErrorDropped(Operators.java:521)                                                                                                                                       │
│     at reactor.core.publisher.EmitterProcessor.onError(EmitterProcessor.java:275)                                                                                                                                │
│     at dev.miku.r2dbc.mysql.client.ReactorNettyClient.drainError(ReactorNettyClient.java:254)                                                                                                                    │
│     at dev.miku.r2dbc.mysql.client.ReactorNettyClient.handleClose(ReactorNettyClient.java:260)                                                                                                                   │
│     at reactor.core.publisher.FluxPeek$PeekSubscriber.onComplete(FluxPeek.java:257)                                                                                                                              │
│     at reactor.core.publisher.Operators$MultiSubscriptionSubscriber.onComplete(Operators.java:1871)                                                                                                              │
│     at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.drain(FluxConcatMap.java:359)                                                                                                                     │
│     at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.onComplete(FluxConcatMap.java:268)                                                                                                                │
│     at reactor.core.publisher.EmitterProcessor.checkTerminated(EmitterProcessor.java:489)                                                                                                                        │
│     at reactor.core.publisher.EmitterProcessor.drain(EmitterProcessor.java:350)                                                                                                                                  │
│     at reactor.core.publisher.EmitterProcessor.onComplete(EmitterProcessor.java:293)                                                                                                                             │
│     at dev.miku.r2dbc.mysql.client.ReactorNettyClient.resumeError(ReactorNettyClient.java:215)                                                                                                                   │
│     at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:88)                                                                                                              │
│     at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:214)                                                                                                                                 │
│     at reactor.netty.channel.FluxReceive.terminateReceiver(FluxReceive.java:424)                                                                                                                                 │
│     at reactor.netty.channel.FluxReceive.drainReceiver(FluxReceive.java:210)                                                                                                                                     │
│     at reactor.netty.channel.FluxReceive.onInboundNext(FluxReceive.java:353)                                                                                                                                     │
│     at reactor.netty.channel.ChannelOperations.onInboundNext(ChannelOperations.java:353)                                                                                                                         │
│     at reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:96)                                                                                                              │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)                                                                                                    │
│     at dev.miku.r2dbc.mysql.client.MessageDuplexCodec.handleDecoded(MessageDuplexCodec.java:187)                                                                                                                 │
│     at dev.miku.r2dbc.mysql.client.MessageDuplexCodec.channelRead(MessageDuplexCodec.java:95)                                                                                                                    │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)                                                                                                    │
│     at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)                                                                                                                │
│     at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)                                                                                                                    │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)                                                                                                    │
│     at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)                                                                                                         │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)                                                                                                  │
│     at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)                                                                                                  │
│     at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)                                                                                                                  │
│     at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792)                                                                                     │
│     at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475)                                                                                                                               │
│     at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)                                                                                                                                        │
│     at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)                                                                                                              │
│     at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)                                                                                                                                 │
│     at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)                                                                                                                     │
│     at java.base/java.lang.Thread.run(Unknown Source)                                                                                                                                                            │
│ Caused by: dev.miku.r2dbc.mysql.client.MySqlConnectionClosedException: Connection unexpectedly closed                                                                                                            │
│     at dev.miku.r2dbc.mysql.client.ClientExceptions.unexpectedClosed(ClientExceptions.java:32)                                                                                                                   │
│     ... 40 common frames omitted                                                                                                                                                                                 │
                                                                                             
```

**Marking as Critical since this is blocking users from using their MySQL datasources and queries without any workaround.**

--- a/app/server/Dockerfile
+++ b/app/server/Dockerfile
@@ -1,6 +1,6 @@
 #When you are building, name it appsmith-server which is how it is referenced in docker-compose.yml
 
-FROM adoptopenjdk/openjdk11:alpine-jre
+FROM adoptopenjdk/openjdk11:jre-11.0.10_9-alpine
 
 LABEL maintainer="tech@appsmith.com"
 

